ext/tiny_tds/extconf.rb

```diff
@@ -22,35 +22,39 @@ def do_help
 
 # Make sure to check the ports path for the configured host
 host = RbConfig::CONFIG['host']
-project_dir = File.join(['..']*4)
+project_dir = File.expand_path("../../..", __FILE__)
 freetds_ports_dir = File.join(project_dir, 'ports', host, 'freetds', FREETDS_VERSION)
 freetds_ports_dir = File.expand_path(freetds_ports_dir)
 
 # Add all the special path searching from the original tiny_tds build
-# order is important here! First in, last searched.
+# order is important here! First in, first searched.
```

Comment:
Curious for my own understanding, why was it this order worked in the past but is now a problem?

Reply:
Previously we used a side effect of calling dir_config multiple times. This doesn't work any longer on Ruby-2.7.

Reply:
I see, thanks for explaining. Is it odd in retrospect that we were calling dir_config in a loop rather than setting up the directories in the right order?

Reply:
I think so.